website/docs/d/domain_record.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+layout: "exoscale"
+page_title: "Exoscale: exoscale_domain_record"
+sidebar_current: "docs-exoscale-domain_record"
+description: |-
+  Provides information about a Domain Record.
+---
+
+# exoscale\_domain\_record
+
+Use this data source to look Domain [Records][record].
+
+[templates]: https://community.exoscale.com/documentation/dns/api/
+
+## Example Usage
+
+The example below matches all Domain Records that match with name `mailserver` and Record type `MX`.
+
+```hcl
+data "exoscale_domain" "exo" {
+  name = my-company.com
+}
+
+data "exoscale_domain_record" "mx" {
+  domain = test.com"
+  filter {
+    name   = "mailserver"
+    recorde_type  = "MX"
+    # OR 
+    # id   = 12345
+    # OR 
+    # record_type  = "MX"
+    # OR 
+    # content  = "mta*"  You can use Regex or not.
```

Comment:
This is not a regexp (at least it is but it'd match what you expect, you probably want `mta.*` instead). Besides, this example is very confusing: I'd suggest breaking it into several smaller simpler examples such as:

```terraform
data "exoscale_domain_record" "mycompany_mailservers" {
  domain = data.exoscale_domain.mycompany.name
  filter {
    name   = "mailserver"
    recorde_type  = "MX"
  }
}

data "exoscale_domain_record" "mycompany_nameservers" {
  domain = data.exoscale_domain.mycompany.name
  filter {
    content  = "ns.*"
  }
}
```